manager/integration/tests/test_basic.py

```diff
@@ -4429,20 +4519,44 @@ def test_space_usage_for_rebuilding_only_volume():  # NOQA
     7. Wait for the rebuilding complete. And verify the volume actual size
        won't be greater than 3x of the volume spec size.
     8. Delete the volume.
+    """
+    # Pre condition
+    prepare_space_usage_for_rebuilding_only_volume(client)
 
-    Case 2: the normal scenario
-    1. Create a new volume with 3Gi spec size.
-    2. Write 3Gi data (using `dd`) to the volume.
-    3. Take a snapshot then mark this snapshot as Removed.
-       (this snapshot won't be deleted immediately.)
-    4. Write 3Gi data (using `dd`) to the volume again.
-    5. Delete a random replica to trigger the rebuilding.
-    6. Wait for the rebuilding complete. And verify the volume actual size
-       won't be greater than 2x of the volume spec size.
-    7. Delete the volume.
+    # The worst scenario
```

Comment:
Think we don't need this since you already mentioned previously: `Test case: worst scenario`.